plugins/analysis/kotlin-plugin/src/test/kotlin/arrow/meta/plugins/analysis/AnalysisTests.kt

```diff
@@ -321,6 +321,21 @@ class AnalysisTests {
     )
   }
 
+  @Test
+  fun `pre-conditions for subjects`() {
+    """
+      ${imports()}
+      @Pre(messages = ["not zero divisor"], formulae = ["(not (= (int other) 0))"], dependencies = [])
+      @Subject(fqName = "kotlin/Int/div")
```

Comment:
I see we are using `kotlin/Int/div`. Shouldn't this use the fqName version that uses "kotlin.Int.div" for the name?

Reply:
I had to change this logic, because the fq-name was ambiguous. For example, you could not tell upfront whether something like `kotlin.collections.arrayOf` was package `kotlin` class `collections`, package `kotlin.collections`... So I've decided to change it to `module/class1/class2/.../member`, which makes it clear for example that `kotlin.collections/arrayOf` is a function within the module, but `kotlin/Int/div` is a function within a class.

Note that I might not have found the good way to obtain a descriptor given the fq-name. If there's one in the Kotlin compiler, I think that would be the good one to use.